integration/tests/karpenter/karpenter_test.go

```diff
@@ -68,7 +73,12 @@ var _ = Describe("(Integration) Karpenter", func() {
 				"--cluster", clusterName,
 				"--kubeconfig", params.KubeconfigPath,
 			)
-			Expect(cmd).To(RunSuccessfully())
+			session := cmd.Run()
+			if session.ExitCode() != 0 {
+				dumpEventsAndLogsForDeployment("karpenter=webhook")
+				dumpEventsAndLogsForDeployment("karpenter=controller")
+			}
```

Comment:
Since the `create cluster` command is failing, shouldn't this code be for the `create cluster` command invocation above, instead of for `utils write-kubeconfig`?

Reply:
Ops, that's true. Well spotted. :)